sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -695,9 +695,9 @@ def from_runner_api(cls,
     # type: (...) -> Optional[PTransform]
     if proto is None or proto.spec is None or not proto.spec.urn:
       return None
-    parameter_type, constructor = cls._known_urns[proto.spec.urn]
 
     try:
+      parameter_type, constructor = cls._known_urns[proto.spec.urn]
```

Comment:
Ditto. Not sure why we need this update given that tests should already pass for Dataflow.

Reply:
It was for fixing Java xlang pipelines but the fix is still incomplete anyway. Will revert this.